Classes/Scheduler/Counter.php

```diff
@@ -202,7 +202,7 @@ private function updateUser() {
 		//Find all users with their current rank
 		$query = 'SELECT fe.uid, fe.tx_typo3forum_rank
 				  FROM fe_users AS fe
-				  WHERE fe.disable=0 AND fe.deleted=0 AND fe.tx_extbase_type="\Mittwald\Typo3Forum\Domain\Model\User\FrontendUser"
+				  WHERE fe.disable=0 AND fe.deleted=0 AND fe.tx_extbase_type="Mittwald\\\Typo3Forum\\\Domain\\\Model\\\User\\\FrontendUser"
```

Comment:
The explanation: Because of the single quotes the backslash will be interpreted as escape character. The content of $query will be

```
SELECT tx_typo3forum_rank, COUNT(*) AS counter FROM fe_users WHERE disable=0 AND deleted=0 AND tx_extbase_type="Mittwald\\Typo3Forum\\Domain\\Model\\User\\FrontendUser" AND pid=16251 GROUP BY tx_typo3forum_rank
```
For MySQL it's the same. The backslash needs to be escaped to be a backslash. The tx_extbase_type does not start with a backslash. Therefore we need to remove the leading backslash to match rows.